### ISSUE TITLE
Error code 65 - Can't load Kernel binary: Invalid kernel binary format version.

### ISSUE THREAD

**Issue by themisir** [axel-op/dart_package_analyzer#14]
The action fails with exit code 65. Here's action logs: https://github.com/hivedb/hive/runs/1798886741

```
2021-01-30T21:49:16.8480259Z Analytics reporting disabled.
2021-01-30T21:49:16.8486736Z 
2021-01-30T21:49:16.8487437Z You may need to restart any open editors for them to read new settings.
2021-01-30T21:49:16.8776796Z ##[endgroup]
2021-01-30T21:49:17.1121542Z ##[group]Running pana
2021-01-30T21:49:17.1840827Z Can't load Kernel binary: Invalid kernel binary format version.
2021-01-30T21:49:17.3124500Z No active package pana.
2021-01-30T21:49:17.3539451Z ##[endgroup]
2021-01-30T21:49:17.3544364Z ##[error]Pana exited with code 65
2021-01-30T21:49:32.8081063Z Cleaning up orphan processes
```

**Comment by creativecreatorormaybenot** [comment]
Same for me: https://github.com/creativecreatorormaybenot/microphone/runs/1802321352?check_suite_focus=true

**Comment by llucax** [comment]
Also affected by this. Is weird because if I try running `pana` inside the docker container `axelop/dart_package_analyzer:v3` it works.

**Comment by axel-op** [comment]
I think that this is a duplicate of #12. Please comment on the previous issue.

**Comment by creativecreatorormaybenot** [comment]
@axel-op It does indeed seem to be the same. This issue seems to be getting more traction, however, so it might be worth keeping this one open instead.

**Comment by axel-op** [comment]
Closed with #12.
